src/utf8_fix.cc

```diff
@@ -57,16 +59,16 @@ char* FixCode(char* b, const char* e, RandomEngine* random) {
       // [0xD800, 0xE000) are reserved for UTF-16 surrogate halves.
       if (c < 0x800 || (c >= 0xD800 && c < 0xE000)) {
         uint32_t halves = 0xE000 - 0xD800;
-        c = std::uniform_int_distribution<char32_t>(0x800,
-                                                    0xFFFF - halves)(*random);
+        c = std::uniform_int_distribution<int32_t>(0x800,
+						    0xFFFF - halves)(*random);
```

Comment:
cpplint.py on Travis: src/utf8_fix.cc:63:  Tab found; better to use spaces  [whitespace/tab] [1]

Reply:
Hmm...I could be horribly wrong but I'm not getting this issue when I run cpplint.py nor do I see it in the travis logs for my changes. 
Maybe you are talking about this PR before I added the commit titled "remove whitespace change"?

Reply:
hm, quite possible. sorry.